security/network_defense.adoc

```diff
@@ -0,0 +1,78 @@
+[[security-network]]
+= Network Defense
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:sourcedir: ../
+
+toc::[]
+
+[[security-network-isolating-applications]]
+== Isolating Applications from Each Other
+
+Deploying modern microservices-based applications in containers often means
+deploying multiple containers distributed across multiple nodes. You need a way
+to isolate applications from other applications within a cluster. You need to
+consider network defense.
+
+A typical public cloud container service, such as Google Container Engine (GKE),
+Azure Container Services, and AWS Container Service, are single tenant services.
```

Comment:
In the future, this may not be true (multi-tennant deployments) may be more common.